src/devtools/client/webconsole/actions/messages.js

```diff
@@ -232,9 +236,13 @@ export function setPauseExecutionPoint(executionPoint, time) {
 }
 
 export function messageOpen(id) {
-  return {
-    type: MESSAGE_OPEN,
-    id,
+  return ({ dispatch, getState }) => {
+    const filtersState = getAllFilters(getState());
+    return dispatch({
+      type: MESSAGE_OPEN,
+      id,
+      filtersState,
```

Comment:
Same thing in this file. Is the `filtersState` being handled by the reducer somewhere?